contributors_guide.rst

```diff
@@ -0,0 +1,376 @@
+
+        """
+	pyart.retrieve.velocity_azimuth_display
+	=======================================
+	
+	Retrieval of VADs from a radar object.
+
+	.. autosummary::
+    	    :toctreeL generated/
+    	    :template: dev_template.rst
+
+	    velocity_azimuth_display
+            _inverse_dist_squared
+            _Average1D
+
+        """
+
+Following the introduction code, modules are then added. Main imports come
+first, followed by 'from imports' and user's own modules.
+
+For example:
+
+.. code-block:: python
+
+        import numpy as np
+        import numpy.ma as ma
+        from scipy.interpolate import interp1d
+
+        from ..core import HorizontalWindProfile
+
+Following the main function def line, but before the code within it, a doc
+string is needed to explain arguments, returns, references if needed, and
+other helpful information.
+
+For example:
+
+.. code-block:: python
+
+        def velocity_azimuth_display(
+            radar, velocity=None, z_want=None, valid_ray_min=16,
+            gatefilter=False, window=2):
+	
+            """
+   	    Velocity azimuth display.
+
+            Parameters
+            ----------
+            radar : Radar
+                Radar object used.
+            velocity : string
+                Velocity field to use for VAD calculation.
+                If None, the default velocity field will be used.
+
+            Other Parameters
+            ----------------
+            z_want : array
+                Height array user would like for the VAD
+                calculation. None will result in a z_want of
+        	np.linspace and use of _inverse_dist_squared
+        	and _Average1D functions. Note, height must have
+        	same shape as expected u_wind and v_wind if user
+        	provides z_want.
+    	    valid_ray_min : int
+        	Amount of rays required to include that level in
+        	the VAD calculation.
+            gatefilter : GateFilter
+        	Used to correct the velocity field before its use
+        	in the VAD calculation. Uses Py-ART's region dealiaser.
+    	    window : int
+        	Value to use for window calculation in _Averag1D
+        	function.
+
+            Returns
+            -------
+    	    height : array
+        	Heights in meters above sea level at which horizontal winds were
+        	sampled.
+    	    speed : array
+        	Horizontal wind speed in meters per second at each height.
+    	    direction : array
+        	Horizontal wind direction in degrees at each height.
+    	    u_wind : array
+        	U-wind mean in meters per second.
+    	    v_wind : array
+        	V-wind mean in meters per second.
+
+    	    Reference
+    	    ----------
+    	    K. A. Browning and R. Wexler, 1968: The Determination
+    	    of Kinematic Properties of a Wind Field Using Doppler
+	    Radar. J. Appl. Meteor., 7, 105–113
+
+    	    """
+
+Private or smaller functions can have a small explanation without in depth
+details. For example:
```

Comment:
Typically a single line is sufficient for private function, and if not a single line intro followed by any multiple lines docs.  Rather than trying to explain this a different example would be better.